test/apps/iodemo/io_demo.cc

```diff
@@ -2041,10 +2041,12 @@ class DemoClient : public P2pDemoCommon {
                         io_op_perf_info[op].min_index = server_index;
                     }
 
-                    total_bytes_completed         += bytes_completed;
-                    total_completed               += delta_completed;
-                    server_info.prev_completed[op] =
+                    total_bytes_completed          += bytes_completed;
+                    total_completed                += delta_completed;
+                    server_info.prev_completed[op]  =
                             server_info.num_completed[op];
+                    server_info.bytes_sent[op]      = 0;
+                    server_info.bytes_completed[op] = 0;
```

Comment:
looks weird: why need both reset bytes_completed to 0 and also keep track of prev_completed?

Reply:
let's remove `prev_completed`?